torch/export/_trace.py

```diff
@@ -1411,6 +1413,7 @@ def _strict_export_lower_to_aten_ir(
 
     return ExportArtifact(
         aten=aten_export_artifact,
+        in_spec=orig_in_spec,
```

Comment:
Does `ExportedProgram.module()` work properly when you have nn module graph inputs? (I guess it now needs to know to unflatten some tensor inputs back into an nn module)